mlx_lm/utils.py

```diff
@@ -504,3 +504,29 @@ def save_config(
     # write the updated config to the config_path (if provided)
     with open(config_path, "w") as fid:
         json.dump(config, fid, indent=4)
+
+
+def common_prefix_len(list1, list2):
```

Comment:
Could you replace the calls to https://github.com/ml-explore/mlx-lm/blob/main/mlx_lm/evaluate.py#L29-L35 with calls to this function instead?